actions/build-pr

```diff
@@ -42,20 +41,18 @@ def add_job():
         cwrhelpers.fail_action('Charm does not provide reference bundle '
                                'and none was provided to action')
 
-    job_name = "pr_charm_{}".format(charm_fname)
+    job_name = "cwr_charm_pr_{}_in_{}".format(charm_fname, bundle_fname)
     job_contents = templating.render(
         source="BuildMyPR/config.xml",
         target=None,
         context={
             "gitrepo": hookenv.action_get("repo"),
             "charm_subdir": hookenv.action_get("charm-subdir"),
             "pushtochannel": hookenv.action_get("push-to-channel") or "",
-            "lpid": hookenv.action_get("lp-id") or "",
-            "charmname": charm_name,
```

Comment:
@ktsakalozos i think this is what was causing 500 errors for @petevg.  You're right in that we need to be better at reporting *why* we got a 500, but I'm pretty sure it was because `charmname` should have been `charm_name`.  Otherwise, the jenkins callback url would give a 404, which somehow got translated to a generic 500.